### PR TITLE
feat: add logic for support create WXP XML Add-in

### PR DESCRIPTION
**Pull request by ZYUN-MSFT**
### Background:
* [PM Spec](https://microsoftapc-my.sharepoint.com/:w:/g/personal/wyun_microsoft_com/EY3ts6-mHX5PiDnjBMvF5ikBecTm2ai9yhUpCtoycPRJJg?e=wwuE1h)
* [ADO Link](https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/26879784)

### Description
This is the implement of the above PM Spec section `create a WXP add-in through VS Code with guidance`

### CODE REVIEW

**Review by jayzhang — DISMISSED** [review]

**Review by hund030 — DISMISSED** [review]

File: packages/fx-core/resource/package.nls.json [hunk 1]
```diff
@@ -365,6 +365,40 @@
   "core.createProjectQuestion.openAiPluginManifest.validationError.missingApiUrl": "Missing URL in \"%s\".",
   "core.createProjectQuestion.openAiPluginManifest.validationError.authNotSupported": "Auth type is not supported. Supported auth type: \"%s\".",
   "core.createProjectQuestion.meArchitecture.title": "Architecture of Search Based Message Extension",
+  "core.createProjectQuestion.officeXMLAddin.bar.title": "Office Add-in",
+  "core.createProjectQuestion.officeXMLAddin.bar.detail": "Creating Office Add-in Project",
+  "core.createProjectQuestion.officeXMLAddin.mainEntry.title": "Office Add-in",
+  "core.createProjectQuestion.officeXMLAddin.mainEntry.detail": "Integration with Outlook, Word, Excel or PowerPoint",
```
Comment (supkasar) [inline packages/fx-core/resource/package.nls.json:371]: Needs a comma before 'or' as we follow American English.

File: packages/fx-core/resource/package.nls.json [hunk 2]
```diff
@@ -365,6 +365,40 @@
...
+  "core.createProjectQuestion.officeXMLAddin.create.title": "Select to create a Word, Excel or PowerPoint Add-in",
```
Comment (supkasar) [inline packages/fx-core/resource/package.nls.json:372]: Needs comma before 'or'.

File: packages/fx-core/resource/package.nls.json [hunk 3]
```diff
@@ -365,6 +365,40 @@
...
+  "core.createProjectQuestion.officeXMLAddin.word.title": "Word Add-in",
+  "core.createProjectQuestion.officeXMLAddin.word.detail": "Integration with Microsoft Word",
+  "core.createProjectQuestion.officeXMLAddin.word.sso.title": "Add-in with single sign on",
```
Comment (supkasar) [inline packages/fx-core/resource/package.nls.json:375]: Make it upper case - Single Sign On

File: packages/fx-core/resource/package.nls.json [hunk 4]
```diff
@@ -365,6 +365,40 @@
...
+  "core.createProjectQuestion.officeXMLAddin.word.sso.detail": "Word add-in with single sign on capabilities",
```
Comment (supkasar) [inline packages/fx-core/resource/package.nls.json:376]: Change it to 'Single Sign On'

File: packages/fx-core/resource/package.nls.json [hunk 5]
```diff
@@ -365,6 +365,40 @@
...
+  "core.createProjectQuestion.officeXMLAddin.word.react.title": "Add-in with React framework",
+  "core.createProjectQuestion.officeXMLAddin.word.react.detail": "Create your Word add-in with React framework",
+  "core.createProjectQuestion.officeXMLAddin.word.create.title": "Create a Word Add-in",
+  "core.createProjectQuestion.officeXMLAddin.excel.title": "Excel Add-in",
+  "core.createProjectQuestion.officeXMLAddin.excel.detail": "Integration with Microsoft Excel",
+  "core.createProjectQuestion.officeXMLAddin.excel.sso.title": "Add-in with single sign on",
```
Comment (supkasar) [inline packages/fx-core/resource/package.nls.json:382]: Change to uppercase - Single Sign On

File: packages/fx-core/resource/package.nls.json [hunk 6]
```diff
@@ -365,6 +365,40 @@
...
+  "core.createProjectQuestion.officeXMLAddin.excel.sso.detail": "Excel add-in with single sign on capabilities",
```
Comment (supkasar) [inline packages/fx-core/resource/package.nls.json:383]: Change to uppercase - Single Sign On

File: packages/fx-core/resource/package.nls.json [hunk 7]
```diff
@@ -365,6 +365,40 @@
...
+  "core.createProjectQuestion.officeXMLAddin.excel.react.title": "Add-in with React framework",
+  "core.createProjectQuestion.officeXMLAddin.excel.react.detail": "Create your Excel add-in with React framework",
+  "core.createProjectQuestion.officeXMLAddin.excel.cf.shared.title": "Excel Custom Functions using a Shared Runtime",
+  "core.createProjectQuestion.officeXMLAddin.excel.cf.shared.detail": "Create an Excel add-in leveraging Custom Functions using a Shared Runtime",
+  "core.createProjectQuestion.officeXMLAddin.excel.cf.js.title": "Excel Custom Functions using a JavaScript-only Runtime",
+  "core.createProjectQuestion.officeXMLAddin.excel.cf.js.detail": "Create an Excel add-in leveraging Custom Functions using a JavaScript-only Runtime",
+  "core.createProjectQuestion.officeXMLAddin.excel.create.title": "Create an Excel Add-in",
+  "core.createProjectQuestion.officeXMLAddin.powerpoint.title": "PowerPoint Add-in",
+  "core.createProjectQuestion.officeXMLAddin.powerpoint.detail": "Integration with Microsoft PowerPoint",
+  "core.createProjectQuestion.officeXMLAddin.powerpoint.sso.title": "Add-in with single sign on",
```
Comment (supkasar) [inline packages/fx-core/resource/package.nls.json:393]: Change to uppercase - Single Sign On

File: packages/fx-core/resource/package.nls.json [hunk 8]
```diff
@@ -365,6 +365,40 @@
...
+  "core.createProjectQuestion.officeXMLAddin.powerpoint.sso.detail": "PowerPoint add-in with single sign on capabilities",
```
Comment (supkasar) [inline packages/fx-core/resource/package.nls.json:394]: Change to uppercase - Single Sign On

**Comment by ZYUN-MSFT** [comment]
Hi @supkasar , the localization strings have been updated in the latest commit, please help to review it.

Thanks. 😊

**Review by jayzhang — APPROVED** [review]

**Review by supkasar — APPROVED** [review]

**Review by JerryYangKai — APPROVED** [review]

**Pull request merged**